project/src/store/action.ts

```diff
@@ -28,3 +29,7 @@ export const findOfferNearby = createAction(Action.FIND_OFFER_NEARBY, (value: nu
 export const findOfferReviews = createAction(Action.FIND_OFFER_REVIEW, (value: number) => ({
   payload: value,
 }));
+
+export const changeSortType = createAction(Action.CHANGE_SORT_TYPE, (value: string) => ({
```

Comment:
```suggestion
export const changeSortType = createAction(Action.CHANGE_SORT_TYPE, (value: string) => ({
```

можно не просто string а type union или enum подсунуть. на пример type Sort = "popular" | "rating" ....